core/trino-web-ui/src/main/resources/webapp-preview/src/api/webapp/api.ts

```diff
@@ -78,6 +78,66 @@ export interface WorkerStatusInfo {
     uptime: string
 }
 
+export interface QueryStats {
```

Comment:
I like these typed response classes 👍🏻

Reply:
luckily the auto completion works too in IDEs